packages/apps/spaces/components/organization/organization-list/OrganizationListColumns.tsx

```diff
@@ -15,14 +15,15 @@ import { TimeToRenewalCell } from '../organization-details/renewal/TimeToRenewal
 const columnHelper =
   createColumnHelper<Omit<Organization, 'lastTouchPointTimelineEvent'>>();
 
-export const columns = [
+export const columns = (tabs?: { [key: string]: string } | null) => [
   columnHelper.accessor((row) => row, {
     id: 'ORGANIZATION',
     cell: (props) => {
       return (
         <OrganizationTableCell
           key={props.getValue().id}
           organization={props.getValue()}
+          activeTab={tabs?.[props.getValue()?.id]}
```

Comment:
Maybe we could read from local storage inside the OrganizationTableCell instead of passing it as an argument from outside?

Reply:
Reading from local storage is costly operation, in case we read inside the cell operation will be called for every cell separately, that is sole reason why I decided to pass that as object from list - to minimize the amount of read operation to 1 per table.

Reply:
ok